app/containers/SafeAreaView.tsx

```diff
@@ -1,32 +1,31 @@
 import React from 'react';
-import { StyleSheet } from 'react-native';
+import { StyleSheet, ViewProps } from 'react-native';
 import { SafeAreaView as SafeAreaContext } from 'react-native-safe-area-context';
 
 import { themes } from '../constants/colors';
-import { withTheme } from '../theme';
+import { useTheme } from '../theme';
 
 const styles = StyleSheet.create({
 	view: {
 		flex: 1
 	}
 });
 
-interface ISafeAreaView {
-	testID?: string;
-	theme?: string;
+interface ISafeAreaView extends ViewProps {
 	vertical?: boolean;
-	style?: object;
 	children: React.ReactNode;
```

Comment:
Should we change this `React.ReactNode` ? As we saw on typescript migration discussion

Reply:
Fixed